### PR TITLE
golly: Add version 4.0

### PR DESCRIPTION
**Pull request by NoraTheGamer**

### CODE REVIEW

**Comment by NoraTheGamer** [comment]
#271

**Comment by NoraTheGamer** [comment]
Any chance for a merge?

File: bucket/golly.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "version": "4.0",
+  "description": " For exploring cellular automata like the Game of Life.",
```
Comment (Calinou) [inline bucket/golly.json:3]:
```suggestion
  "description": "For exploring cellular automata like the Game of Life.",
```

File: bucket/golly.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  "homepage": "http://golly.sourceforge.net/",
+  "hash": "cd789caaf7422af9262a13ee3d6235363a05566ce1607a4f1925e6e26dbd84c5",
+  "url": "http://downloads.sourceforge.net/project/golly/golly/golly-4.0/golly-4.0-win-64bit.zip",
+  "license": "Proprietary",
```
Comment (Calinou) [inline bucket/golly.json:7]:
```suggestion
  "license": "GPL-2.0-or-later",
```

File: bucket/golly.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{
+  "version": "4.0",
+  "description": "Program for exploring cellular automata such as the Game of Life",
+  "homepage": "http://golly.sourceforge.net/",
+  "hash": "cd789caaf7422af9262a13ee3d6235363a05566ce1607a4f1925e6e26dbd84c5",
+  "url": "http://downloads.sourceforge.net/project/golly/golly/golly-4.0/golly-4.0-win-64bit.zip",
+  "license": "GPL-2.0-or-later",
+  "pre_install": ["Move-Item -Path $dir/$app-$version-win-64bit/* -Destination $dir", "Remove-Item $dir/$app-$version-win-64bit"],
```
Comment (Calinou) [inline bucket/golly.json:8]:
```suggestion
  "pre_install": [
    "Move-Item -Path $dir/$app-$version-win-64bit/* -Destination $dir",
    "Remove-Item $dir/$app-$version-win-64bit"
  ],
```

**Review by Calinou — APPROVED** [review]
Thanks!

**Pull request merged**